v2/googlecloud-to-snowflake/src/main/java/com/google/cloud/teleport/v2/snowflake/templates/PubSubToSnowflake.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright (C) 2021 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.snowflake.templates;
+
+
+import static com.google.cloud.teleport.v2.snowflake.util.SimpleMapper.getUserDataMapper;
+
+import com.google.cloud.teleport.v2.snowflake.options.PubSubToSnowflakeOptions;
+import com.google.cloud.teleport.v2.snowflake.util.DecryptUtil;
+import com.google.common.base.Strings;
+import java.io.IOException;
+import org.apache.beam.sdk.Pipeline;
+import org.apache.beam.sdk.PipelineResult;
+import org.apache.beam.sdk.io.gcp.pubsub.PubsubIO;
+import org.apache.beam.sdk.io.snowflake.SnowflakeIO;
+import org.apache.beam.sdk.options.PipelineOptionsFactory;
+import org.apache.beam.sdk.options.ValueProvider;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * The {@link PubSubToSnowflake} is a Streaming pipeline which ingests CSV format messages
+ * from Pub/Sub subscription and outputs them into a Snowflake table.
+ *
+ */
+public class PubSubToSnowflake {
+
+    private static final Logger LOG = LoggerFactory.getLogger(PubSubToSnowflake.class);
+
+	/**
```

Comment:
Can you check code formatting?
https://github.com/GoogleCloudPlatform/DataflowTemplates#formatting-code